autoscale/src/main/resources/schema/app/20211220140443_CB-15139_Support_for_StopStart_scaling_in_periscope.sql

```diff
@@ -0,0 +1,12 @@
+-- // CB-15139 Support for StopStart scaling in periscope
+-- Migration SQL that makes the change goes here.
+ALTER TABLE "cluster" ADD COLUMN IF NOT EXISTS "stop_start_enabled" boolean;
+ALTER TABLE "cluster" ALTER COLUMN "stop_start_enabled" SET DEFAULT false;
+UPDATE "cluster" SET "stop_start_enabled" = false WHERE "stop_start_enabled" IS NULL;
```

Comment:
Can we treat NULL values as false? Instead of ALTERING every row in the table?

Reply:
Yes, updated to a simpler schema